DOCS-121: Update Storage/Disks options (#1680)
freenas/freenas-docs: userguide/storage.rst

   +------------------------------+-----------+------------+--------------------------------------------------------------------------------------------------------------------------+
   | Informational                | string    |            | Report if drive temperature is at or above this temperature in Celsius. :literal:`0` disables the report.                |
   +------------------------------+-----------+------------+--------------------------------------------------------------------------------------------------------------------------+

>    | SED Password                 | string    |            | Enter the disk password. Use this field to set or change the password for SED. Leave this field blank to keep a          |

Reviewer: This is a bit wordy and confusing.
```
Set or change the password of this SED. This password will be used instead of the global SED password in :menuselection:`...`
```

   |                              |           |            | previously-set password. This password will be used instead of the global SED password which is set in                   |
   |                              |           |            | :menuselection:`System --> Advanced`. See :ref:`Self-Encrypting Drives`.                                                 |
   +------------------------------+-----------+------------+--------------------------------------------------------------------------------------------------------------------------+

>    | Clear SED Password           | checkbox  |            | Clear a previously-set password.                                                                                         |

Reviewer: ```suggestion
   | Clear SED Password           | checkbox  |            | Clear the SED password for this disk.                                                                                         |
```
